packages/venia-concept/src/util/createTestInstance.js

```diff
@@ -0,0 +1,13 @@
+import TestRenderer, { act } from 'react-test-renderer';
+
+export default (...args) => {
+    let instance;
+
+    // wrap rendering code in `act()`
+    // https://reactjs.org/docs/test-utils.html#act
+    act(() => {
+        instance = TestRenderer.create(...args);
+    });
+
+    return instance;
+};
```

Comment:
I feel like I've seen this before... 😄 

Any way to share it from Peregrine?

Reply:
Yeah, I suppose we could put it in peregrine. For some reason, I'm not totally comfortable having Venia rely on a test utility from another package, but I suppose we already technically do that with `jest`, `react-test-renderer`, etc.

Reply:
@jimbo Is it safe to delete this utility since it's in Peregrine now?